mungegithub/mungers/submit-queue.go

```diff
@@ -947,9 +951,11 @@ func (sq *SubmitQueue) validForMergeExt(obj *github.MungeObject, checkStatus boo
 		}
 	}
 
-	// Clearly
-	if !(obj.HasLabel(lgtmLabel) || obj.HasLabel(approvedLabel)) {
-		sq.SetMergeStatus(obj, noLGTM)
+	if sq.GateApproved && (!obj.HasLabel(lgtmLabel) || !obj.HasLabel(approvedLabel)) {
```

Comment:
This would be much easier for me to read if you just made it 2 checks. And you have 2 consts above.
`noLGTM` and `noApproved`

We recheck (in the SQ) if the code was updated since the LGTM but before the other munger removed the label. Should we do the same here?